src/contexts/ClientContext.tsx

```diff
@@ -1,9 +1,13 @@
-import { CosmWasmClient } from "@cosmjs/cosmwasm";
+import { CosmWasmClient as LaunchpadClient } from "@cosmjs/cosmwasm";
+import { CosmWasmClient as StargateClient } from "@cosmjs/cosmwasm-stargate";
+import { Registry } from "@cosmjs/proto-signing";
 import React from "react";
 
 export interface ClientContextValue {
   readonly nodeUrl: string;
-  readonly client: CosmWasmClient;
+  readonly launchpadClient: LaunchpadClient;
+  readonly stargateClient?: StargateClient;
+  readonly registry: Registry;
```

Comment:
What about something like this, to add more info in the explorer app context?

```suggestion
  readonly typeRegistry: Registry;
```